core/openapi/ogcapi-records-1.yaml

```diff
@@ -237,7 +237,7 @@ components:
                 default: 'http://www.opengis.net/def/uom/ISO-8601/0/Gregorian'
       landingPage:
         allOf:
-          - $ref: "#/components/schemas/common/link.yaml"
```

Comment:
This should be `#/components/schemas/common/link`.  It's supposed to be an internal reference.

Reply:
When do we use internal references vs. `$ref`?  I see both in this file.